### ISSUE TITLE
Cant change Category 

### ISSUE THREAD

**Issue by Tealk144** [fcturner/passwords#184]
Hi, i have the problem that i cant Change the Password Category after i have create the Password.
I have try it on Firefox and Vivaldi.

**Comment by fcturner** [comment]
Thank for letting me know. 

Did you use the category button on the password row? If you refresh the page, does it work then?

**Comment by fcturner** [comment]
Will be solved with #168. Now you need to reload the page (sometimes). It's because of ownCloud uses a specific system for getting values out of a database (called Handlebars).

**Comment by Tealk144** [comment]
Ok if i use the category button on the left side of the edit button,
i get a window there i can set up a category or creat a new one,
there i select a category, and klick on save,
the window remains open and does not save the category

if i select a category at the moment who i create a password, this run fine

refreshing page and use other browser does not help

**Comment by IzK666** [comment]
Maybe it's too late, but If that could help would be great: This error (can't change categories or edit notes) happens when you haven't shared the password.
It throws an error on script.js line 131. I have checked the variable $row.attr(attr_sharedwith) and returns 'undefined'.
I have replaced the if sentence with this code and works again:

				if ($.isArray(sharewith)) {
					sharewithArr = sharewith;
				} else if (!sharewith) {
					sharewithArr = "";
				} else {
					sharewithArr = sharewith.split(', ');
				}

 
Hope that it helps.
